src/store/gateway/reactions.ts

```diff
@@ -147,20 +147,21 @@ export function gatewayReactions(store: Store<DashboardState>) {
   async function checkIncompleteTransfers() {
 
     const plasmaGateways = PlasmaGateways.service()
+    const loomGateway = plasmaGateways.ethereumLoomGateway
+    const mainGateway = plasmaGateways.ethereumMainGateway
+    const loomReceipt = await loomGateway.withdrawalReceiptAsync(loomGateway.caller)
+    const mainReceipt = await mainGateway.withdrawalReceiptAsync(mainGateway.caller)
+    if (!loomGateway && !mainReceipt) return
```

Comment:
Did you mean loomReceipt instead of loomGateway?

Reply:
🤦‍♂️